build-system/tasks/e2e/index.js

```diff
@@ -192,7 +192,7 @@ async function runTests_() {
 async function runWatch_() {
   const filesToWatch = argv.files ? getFilesFromArgv() : config.e2eTestPaths;
 
-  log('Watching', cyan(filesToWatch), 'for changes...');
+  log('Watching', cyan(filesToWatch.join(', ')), 'for changes...');
```

Comment:
I believe this is the intended functionality but the function signature of `cyan` claims to not accept an array as input.

Reply:
That makes sense, will do